src/connection/host_lookup.c

```diff
@@ -36,40 +36,49 @@
 static pthread_mutex_t gethostbyname_lock = PTHREAD_MUTEX_INITIALIZER;
 #endif /* HAVE_GETHOSTBYNAME_R */
 
-
 #define HOSTNAME_BUF_SIZE              (256)
+#define MAX_HOSTS_LINE_NUM       (256)
+#define IPADDR_LEN              (17)
+
+typedef enum
+{
+  HOSTNAME_LOOKUP = 0,
+  IPADDRESS_LOOKUP = 1,
+} LOOKUP_STATUS;
```

Comment:
It is not status, it is category or command or something.
Following is the sample,

```
typedef enum
{
  HOSTNAME_TO_IPADDR = 0,
  IPADDR_TO_HOSTNAME
} HOST_LOOKUP_CMD;
```

Reply:
Yes, I will change it to the type. Thank you!